create_galaxy_tests.sh

```diff
@@ -1,3 +1,3 @@
-conda create -y --quiet --override-channels --channel iuc --channel conda-forge --channel bioconda --channel defaults --name ctdopts-1.3 ctdopts=1.3 lxml
+conda create -y --quiet --override-channels --channel iuc --channel conda-forge --channel bioconda --channel defaults --name ctdopts-1.4 ctdopts=1.4 lxml
```

Comment:
It would be nice to have some line wrapping in this file. ~100 is fine, but makes reviewing annoying if I have to scroll horizontally in GitHub :-)